tests/ctst/steps/azureArchive.ts

```diff
@@ -471,6 +472,17 @@ Then('object {string} should expire in {int} days', async function (this: Zenko,
     assert(diff >= days && diff < days + 0.1);
 });
 
+Given('that lifecycle is {string} for the {string} location',
+    async function (this: Zenko, status: string, location: string) {
+        let path: string;
+        if (status === 'paused') {
+            path = `/_/lifecycle/pause/${location}`;
+        } else {
+            path = `/_/lifecycle/resume/${location}`;
+        }
+        await axios.post(`http://${this.parameters.BackbeatApiHost}:${this.parameters.BackbeatApiPort}${path}`);
```

Comment:
why are we using `axios` here: aren't we usually using simple http request?
(I can see axios used indeed, but seems to be related to hooking aws/s3 requests in some way....)